### PR TITLE
Update JSX page

### PR DESCRIPTION
**Pull request by mattdamon108**
Follow up the issue https://github.com/rescript-association/rescript-lang.org/issues/589

### CODE REVIEW

File: pages/docs/manual/latest/jsx.mdx [hunk 1]
```diff
@@ -108,153 +106,132 @@ React.createElement(React.Fragment, undefined, child1, child2);
...
+React.createElement(Comp, {
+  a: "a",
+  b: "b"
+});
 ```
 
 </CodeTab>
 
-This is extra useful in the cases where you are handled `myChild` that is already a list of things, and want to forward that without wrapping it an extra time (which would be a type error) \*. It also allows you to pass arbitrary data structures at `children` position (remember, JSX `children` is really just a totally normal prop):
+The multiple spreads are not allowed:
```
Comment (fhammerschmidt) [inline pages/docs/manual/latest/jsx.mdx:186]: "Multiple spreads are not allowed:" 
(ditch the `The`)

File: pages/docs/manual/latest/jsx.mdx [hunk 2]
```diff
@@ -108,153 +106,132 @@ React.createElement(React.Fragment, undefined, child1, child2);
...
 ```
 
 </CodeTab>
 
-## Usage
-
-See [ReasonReact JSX](https://reasonml.github.io/reason-react/docs/en/jsx) for an example application of JSX, which transforms the above calls into a ReasonReact-specific call.
-
-Here's a JSX tag that shows most of the features.
+The spread must be the first position followed by other props:
```
Comment (fhammerschmidt) [inline pages/docs/manual/latest/jsx.mdx:196]: "The spread must be **at** the first position, followed by other props:"

**Review by fhammerschmidt — CHANGES_REQUESTED** [review]

**Comment by mattdamon108** [comment]
@fhammerschmidt Thanks! I've fixed all.

**Review by fhammerschmidt — APPROVED** [review]

**Comment by ryyppy** [comment]
looks good. shall we merge?

**Pull request merged**